build.gradle

```diff
@@ -125,7 +125,7 @@ apply from: file('gradle/java/jar-manifest.gradle')
 
 // Publishing and releasing
 apply from: file('gradle/maven/defaults-maven.gradle')
-apply from: file('gradle/releasing.gradle')
+// TODO: apply from: file('gradle/releasing.gradle')
```

Comment:
I'll just remove all trace of releasing.gradle - there don't seem to be a need to add another gradle file here?